### PR TITLE
Refs #15667 -- Added resetting default renderer when FORM_RENDERER is changed.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

**Comment by felixxm** [comment]
I'd like to merge it after fixing ticket-34878 (to keep backport of 34878 straightforward).

**Review by David-Wobrock — APPROVED** [review]
Looks good to me 💯

File: docs/topics/testing/tools.txt [hunk 1]
```diff
@@ -1502,11 +1502,17 @@ Overridden settings               Data reset
...
 TEMPLATES                         Template engines
+FORM_RENDERER                     Default renderer
 SERIALIZATION_MODULES             Serializers cache
 LOCALE_PATHS, LANGUAGE_CODE       Default translation and loaded translations
 STATIC_ROOT, STATIC_URL, STORAGES Storages configuration
 ================================= ========================
 
+.. versionchanged:: 5.1
+
+    Resetting the default renderer when the ``FORM_RENDERED`` setting is
```
Comment (kezabelle) [inline docs/topics/testing/tools.txt:1513]: Typo?
```suggestion
    Resetting the default renderer when the ``FORM_RENDERER`` setting is
```

Comment (felixxm) [reply docs/topics/testing/tools.txt:1513]: Good catch :dart:

**Pull request merged**